### ISSUE TITLE
Can't scroll when too many groups added.

### ISSUE THREAD

**Issue by Tylaw93** [JasonCostanza/Dice-Vault#52]
**Describe the bug**
When you add more than 6 groups to a roll you can't scroll down to pin or save the roll.

**To Reproduce**

**Expected behavior**
I would have expected the whole page to be scrollable or for them to function as two separate scrollable screens. 

**Additional context**
I know this is niche but I believe should be an easy fix. I use it to roll a creatures full round of attacks speeding up combat.

**Screenshots**
![Screenshot 2024-09-23 103551](https://github.com/user-attachments/assets/1e7e7ba4-77b0-43e6-a4f7-a7c06f164bc2)

**Comment by JasonCostanza** [comment]
I'll look into making this scrollable when I can. I didn't assume anyone would actually use this many groups. Looking at the above though you're rolling attack rolls _and_ damage at the same time. Do you just roll everything regardless whether it hits or misses? I guess my point here is if the War Chief is not using his Spear why clutter your rolls with Spear damage? I'll still look into making it scroll however. :)

**Comment by Tylaw93** [comment]
Yes I roll everything regardless if it hits or misses in larger combats, one of those small time saves that adds up over time. I agree in this used screenshot the spear damage is redundant because typically you would be rolling either ranged or close combat, but the problem still is present if an attacker had more 2 attack actions that you wanted to roll at once. I appreciate you responding so quick and whether this gets addressed or not I love the work you guys have done, makes running my games way smoother.

**Comment by JasonCostanza** [comment]
@Tylaw93 If you're up to the task, would you like to test my changes in the release/3.1 branch? It should have enabled scroll bars for the staged roll at the top and then unified the appearance of the scroll bars for the pinned rolls down below. It supports I believe roughly 5 groups before it switches into a scrolling form factor. Let me know how that works.

**Comment by JasonCostanza** [comment]
I fixed this but I don't remember when. Somewhere in 3.1 iirc.
